docs/courses/csintro1/review/baseball-catching.md

```diff
@@ -16,12 +16,12 @@ Start the project, and create a mitt for the player to control.
 
 ## Student Task #2: Throwing Balls
 
-Add balls for the player to attempt to collect
+Add balls for the player to attempt to collect.
 
 1. Get an ``||game:on game update every 500 ms||`` event
-2. Create a projectile representing a baseball inside the ``||game:on game update every||`` event
-3. Change the variable name from ``||variables:projectile||`` to ``||variables:baseball||``, and the ``||sprites:kind||`` from ``Player`` to ``Baseball``
-4. Making ``||variables:baseball||`` move randomly in the ``x`` and ``y`` directions using ``||math:pick random 0 to 10||`` for the initial ``||sprites:vx||`` and ``||sprites:vy||``: make the range from `-50` and `50`
+2. Create a ``||sprites:projectile from side||`` representing a baseball inside the ``||game:on game update every||`` event and change the variable name from ``||variable:projectile||`` to ``||variable:baseball||``
```

Comment:
This line is using variable and the next is variables for the inline snippet - i think variables is the right one?